interfaces/Portalicious/src/app/components/page-layout/page-layout.component.html

```diff
@@ -4,7 +4,7 @@
     @if (programId()) {
       <app-program-header [programId]="programId()!" />
     }
-    <div class="px-6 py-4 lg:px-24">
+    <div class="mx-auto max-w-screen-xl px-6 py-4">
```

Comment:
This is the change I was discussing this morning in the call (ie. the website not being full screen anymore). I will revert this in a separate PR.